pebbles/views/blueprints.py

```diff
@@ -91,14 +91,14 @@ def post(self):
         if not user.is_admin and not is_group_manager(user, group):
             logging.warn("invalid group for the user")
             abort(403)
-        user_owned_blueprints = Blueprint.query.filter_by(group_id=group_id).count()
+        user_owned_blueprints = Blueprint.query.filter_by(group_id=group_id, current_status='active').count()
```

Comment:
I wonder if we could make use of the global access control here. Like get all blueprints that user has access to, and then in the second pass select those that user owns.

Reply:
Looks good now.